File: angular/src/shared/Models/EmailRequest.ts

export class EmailRequest {
    public toEmail: string | undefined = undefined;
    public subject: string | undefined = undefined;
    public body: string | undefined = undefined;
    public eventName: string | undefined = undefined;
    public date: Date | undefined = undefined;
    public eventAddress: string | undefined = undefined;
    public eventImage: string | undefined=undefined;
    public guestId: number| undefined=0;
    public eventId: number| undefined=0;
}

File: aspnet-core/src/Event_Planning_System.Application/Email/EmailRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Event_Planning_System.Email
{
    public class EmailRequest
    {
        public string ToEmail { get; set; }
        public string Subject { get; set; }
        public string Body { get; set; }
        public string EventName { get; set; }
        public DateTime Date { get; set; }
        public string EventAddress { get; set; }
        public string EventImage { get; set; }
        public int? GuestId { get; set; }
        public int? EventId { get; set; }

    }
}
